### PR TITLE
change(eosio-core): make adjustments

### LINKED ISSUE #261: check getting started page
Content seems too short and incomplete

https://guide.eoscostarica.io/docs/tutorials/getting-started

### LINKED ISSUE #266: Change EOSIO Core

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #266  
Resolves #261 

### Steps to test
1. EOSIO Core adjustments

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — DISMISSED** [review]
LGTM

**Review by steph222 — DISMISSED** [review]
Good Work!!

**Review by xavier506 — APPROVED** [review]
LGTM

File: sidebars.js [hunk 1]
```diff
@@ -32,14 +32,13 @@ module.exports = {
...
         'eos-learn/ricardian-contracts',
-        'eos-learn/eosio-core'
+        'eos-learn/eosio-components'
       ],
     },
     {
       type: 'category',
       label: 'Tutorials',
       items: [
-        'tutorials/getting-started',
```
Comment (leisterfrancisco) [inline sidebars.js:42]: Is it right?

File: docs/eos-learn/eosio-components.md [hunk 2]
```diff
@@ -53,6 +63,7 @@ This installation guide will aim for Windows users, unfortunately, EOSIO does no
 - Now you need to restart your Windows OS
 - Finally , go to your *Microsoft Store* and Search for whatever Linux OS you want, we recommend you install Ubuntu 18.04 LTS
```
Comment (leisterfrancisco) [inline docs/eos-learn/eosio-components.md:64]: > Finally ,

Extra space

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (xavier506) [reply sidebars.js:42]: yes

File: docs/eos-learn/eosio-components.md (hunk 2 above)
Comment (xavier506) [reply docs/eos-learn/eosio-components.md:64]: we can use the 👾 Space Invader 👾 emoji when we find an extra space that needs to be removed

**Review by xavier506 — APPROVED** [review]

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Pull request merged**